doc/usage.rst

```diff
@@ -1,33 +1,55 @@
-Usage
-=====
+Using imSim
+===========
 
-To run the example config, change into the *imSim* source directory,
-then
+imSim is implemented as a set of modules that are executed by the *GalSim* program.  In order to run *imSim* you create a YAML file which imports and configures the modules.  *imSim* builds on basic *GalSim* functionality so when using *imSim* you will configure both *GalSim* and *imSim* functionality.  In this section a simple config file is demonstrated.  You can find a more complete description of the config system option in the :doc:`Config System <config>`.
+
+*imSim* is packaged with some YAML config files for it's own use which you can look at if you would like to see some examples.  You can see some in the "*config*" and also the "*examples*" directory.
+
+To get started copy the two files ``imsim-user.yaml`` and ``example_instance_catalog.txt`` to a working directory.  Edit the ``imsim-user.yaml`` file to point to your local copy of the ``example_instance_catalog.txt`` file by removing the two instances of ``$os.environ.get('IMSIM_HOME')+`` from the file.
```

Comment:
I'd thought our default suggested usage was going to switch over to using skycatalog.  So I thought we'd want our getting started instructions to use that.  You mention it below, but it's not very clear to a beginner user how one would get going with using skycatalog.

Reply:
Well... for pedagogical purposes alone, I want people to be able to create, see, and modify what is in the file they are running in this usage section.  We currently don't even  have a way for users to generate a skycatalog.   So, I would like to keep this. 

My intention when we have more information on skycatalogs and how to create and use them is to add information to the (currently empty) data-formats.rst. 

But, more general discussion of what goes in the config yaml etc skycatalogs etc... I will move to the issue you opened since this PR is so messed up.